src/pages/api/bookmarks.ts

```diff
@@ -1,5 +1,6 @@
 import type { NextApiRequest, NextApiResponse } from 'next'
 
+// eslint-disable-next-line import/no-extraneous-dependencies
```

Comment:
`eslint-plugin-import` is complaining about these imports not being defined in package.json but they are not external imports. This will become a moot point when embedded Keystone is removed.